data/boundaries/wilderness/index.html

```diff
@@ -54,8 +54,9 @@ <h3 id="WildernessBoundaries">Wilderness Boundaries</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{% link images/wilderness-boundaries-ut.jpg %}" alt="Wilderness" />
-    <p>Boundaries of federally designated wilderness on both Forest Service and BLM lands. These boundaries are a subset of the statewide <a href="{% link data/cadastre/land-ownership/index.html %}">Land Ownership</a> layer and are attributed as <code>"desig"="Wilderness"</code>. Go find some wild places!<p>
-    <p>The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 meter coordinate precision). There are no constraints or warranties with regard to the use of this dataset. Users are encouraged to attribute content to: State of Utah, SGID. The dataset is maintained by the State Institutional Trust Lands (SITLA) in cooperation with the BLM and U.S. Forest service.</p>
+    <p>This data represents the combined BLM and USFS wilderness areas of Utah. Wilderness areas from the BLM National Lands Conservation System (NLCS) were merged with U.S. Forest Service wilderness areas in the National Wilderness Preservation System, preserving the polygon geometry and the name attributes only. The complete BLM and USFS data can be found at:</p>
+    <p>https://www.blm.gov/services/geospatial/GISData/utah</p>
```

Comment:
Would you please

1. join the paragraph tags into one 
1. Add a href tags to the links
1. And if you want the links on separate lines use a ul/li tags?